gradle/libs.versions.toml

```diff
@@ -1,12 +1,12 @@
 [versions]
-agp = "7.3.1"
+agp = "8.0.0"
 grpc = "1.47.0"
-kotlin = "1.7.22"
+kotlin = "1.8.22"
 espresso = "3.4.0"
 androidxLifecycle = "2.5.1"
 androidxTest = "1.5.0"
 moshi = "1.14.0"
-okhttp = "4.10.0"
+okhttp = "4.11.0"
```

Comment:
okhttp = "4.11.0"  version fixed the build release problem with R8. 
https://github.com/square/okhttp/issues/7777